tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
-envlist = py26, py27, py33, py34, py35
+envlist = py27,py33,py34,py35
 
 [testenv]
 deps =
-    mock
     pyzmq
-    unittest2
+    pytest
+    py27: mock
 commands =
-    unit2 discover []
+    pytest {posargs}
```

Comment:
pytest is great... but nothing in this PR requires this change.   And more importantly... We don't just arbitrarily change the dev/test environment without discussion.

thanks for removing this in your rebase.